compliance/azure/azure_untagged_resources/untagged_resources.pt

```diff
@@ -271,21 +311,21 @@ policy "policy_azure_resource" do
       field "id" do
```

Comment:
Line 287 - could change to something like this for `summary_template` :

`"{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Azure untagged resources found."`